test/eventstream-test.js

```diff
@@ -0,0 +1,129 @@
+'use strict;'
+
+const expect = require('chai').expect
+const testRoot = require('path').resolve(__dirname, '..')
+const glp = require('bindings')({ module_root: testRoot, bindings: 'glpk' })
+const temp = require('temp').track()
+const fs = require('fs')
+
+glp.termOutput(false)
+
+describe('Verify eventemitter on Problem object', function() {
+    it('should have log events fired as the problem is processed asynchronously', function(done) {
+        let lp = new glp.Problem()
+        let idx = 0;
+        let messages = [
+            "GLPK Integer Optimizer",
+            "0 rows, 0 columns, 0 non-zeros",
+            "0 integer variables, none of which are binary",
+            "Preprocessing...",
+            "Objective value =   0.000000000e+00",
+            "INTEGER OPTIMAL SOLUTION FOUND BY MIP PREPROCESSOR",
+        ]
+        lp.on('log', function(msg) {
+            expect(msg).to.contain(messages[idx++])
+            if(idx === messages.length) {
+                done()
+            }
+        })
+        lp.intopt({ msgLev: glp.MSG_ALL, presolve: glp.ON }, function() {
```

Comment:
It is for the purposes of logging. It doesn't actually exercise the presolver though because the lp is empty